specs/StopServers.groovy

```diff
@@ -96,6 +96,55 @@ class StopServers extends PluginTestHelper {
         runCliCommand(CliCommandsGeneratorHelper.removeServerGroupCmd(serverGroup))
     }
 
+    @Unroll
+    def "StopServers, group with all servers started on a different hosts (C277751)"() {    //shutdown host. After test need start server
```

Comment:
let's put ignored mark on for such tests (different hosts). I will merge it to master now, let's change it there